cppcache/src/ClientMetadataService.cpp

```diff
@@ -143,7 +143,12 @@ void ClientMetadataService::getClientPRMetadata(const char* regionFullPath) {
                                               reply.getFpaSet());
       if (m_bucketWaitTimeout > std::chrono::milliseconds::zero() &&
           reply.getNumBuckets() > 0) {
-        WriteGuard guard(m_PRbucketStatusLock);
+        std::unique_lock<std::timed_mutex> guard(m_timedBucketStatusLock,
+                                                 std::chrono::seconds(60));
```

Comment:
Why 60 seconds here where all the other uses is 1 second and uses a `constexpr`?

Reply:
Other uses  come in the context of  read/white requests , that has their own timeout,  beyond  which  as I understand the is no point to  continue waiting  for bucket.  I think now,  maybe it is better I will  use  read/write timeout constants here as basis  for mutex timeout , instead of defining new ones?  

Here specifically  though,  the flow is stopping the service for the pool, so I think the timeout can be higher , that is why defined 1 minute